content/spin/v3/community-kinesis-trigger.md

```diff
@@ -0,0 +1,131 @@
+title = "The Spin Kinesis Trigger (Community)"
```

Comment:
Not sure this needs to be in the title.  It might work better in a boxout at the top of the page where you can explain the significance a la "not maintained by Fermyon" etc.